subsys/bluetooth/services/cts.c

```diff
@@ -0,0 +1,192 @@
+/** @file
+ *  @brief GATT Current Time Service
+ */
+
+/*
+ * Copyright (c) 2024 Croxel Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdbool.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/posix/time.h>
+
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/services/cts.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(cts, CONFIG_BT_CTS_LOG_LEVEL);
+
+#define BT_CTS_ATT_ERR_VALUES_IGNORED 0x80
+#define BT_CTS_FRACTION_256_MAX_VALUE 255
+
+static const struct bt_cts_cb *_cb;
+
+#if CONFIG_BT_CTS_HELPER_API
```

Comment:
This should be #ifdef and not #if. If the Kconfig option is not enabled the macro will not be defined at all.